docs/user-guides/userguide_linux.md

```diff
@@ -0,0 +1,69 @@
+# Usage on Linux/Unix
+
+## Mumble (Client)
+
+### Installation
+
+The best option to install Mumble is through your distribution's package repository.
```

Comment:
```suggestion
The best option to install Mumble is through your distribution's package manager.
```
Usually it's either "package manager" or "repository"

Reply:
I think `repository` is the word to use, because there can be many different package managers.
As far as I remember I only copied the wording "package repository" and while some people might think it is too much, it is not wrong in any way.

Still I can use "repository" only, if you insist.

Reply:
In all contexts I ever read about this it was either the one or the other. I think I never encountered "package repository". SO I guess I'd prefer simply "repository" in this case :thinking:

Reply:
> I think I never encountered "package repository"

It is used very commonly, for example the `Arch package repository` is named like that.

I changed it to repository, users will likely know what it means.